workshop/content/docs/api-deep-dive/maps.md

```diff
@@ -20,11 +20,18 @@ title: OGC API - Maps
 
 ## Introduction
 
-The draft OGC API - Maps standard describes an API that presents data as maps by applying a
+The OGC API - Maps standard describes an API that presents data as maps by applying a
```

Comment:
`[OGC API - Maps](https://ogcapi.ogc.org/maps) is a specification that describes...`